gulpfile.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+// This file is only for generating the docs
+// No need to use any of this if working on the main website
+
+const { src, dest, series } = require('gulp');
+const pump = require('pump');
+const through2 = require('through2');
+const frontMatter = require('gray-matter');
+const download = require('github-download-directory');
+const flatten = require('gulp-flatten');
```

Comment:
This shouldn't be necessary.

Reply:
@SomethingNew71 wow, docusuarus is very opinionated about directory structure and stuff.  I've updated my `pull-docs` branch with changes that avoid this problem (specifically https://github.com/gulpjs/gulpjs.github.io/commit/79f0f9817fab2c1e47cccfbad2f3628016ebfa28) - with these changes, you need to add `customDocsPath: 'converted-docs'` to the siteConfig.js

Reply:
Yeah no joke, they want it one way or its the highway. Anyways, sounds good I'll get that added.